### DISCUSSION TITLE
Can I connect pipelines like I connect solids?

### DISCUSSION THREAD

**Discussion by wangm23456** [dagster-io/dagster#3543] [Ideas]
Can I connect pipelines like I connect solids? I think this will Improve efficiency for my coding life.

**Comment by alangenfeld** [comment]
Not yet, but we are working on it, see https://github.com/dagster-io/dagster/discussions/2902

**Comment by sryza** [comment]
Hi @wangm23456 - have you explored composite solids?  Are there limitations that make it difficult to use them for what you're trying to do?

**Reply by wangm23456** [reply]
Cumbersome.

**Reply by ivanov-slk** [reply]
You can check [https://stackoverflow.com/questions/66672656/dagster-start-pipeline-from-another-pipeline-using-its-outputs/](url) for more ideas. I'd vote up for the Asset sensor, if this works for you. Otherwise... for now you might need to stick with the cumbersome approach.
